src/formatterLoader.ts

```diff
@@ -81,15 +80,14 @@ function loadFormatter(
 function loadFormatterModule(name: string): FormatterConstructor | undefined {
     let src: string;
     try {
-        // first try to find a module in the dependencies of the currently linted project
-        src = resolve.sync(name, { basedir: process.cwd() });
```

Comment:
you basically just need to pass `preserveSymlinks: false` to `resolve`

This will be the default behavior in next major version of resolve, by the way

Reply:
We need to try both the resolved and the unresolved path because TSLint relies on both, so I think the way I made this change is the simplest approach.